client/task-list/tasks/payments/components/WCPayMethodCard.js

```diff
@@ -37,26 +39,18 @@ const TosPrompt = () =>
 		},
 	} );
 
-const ButtonComponent = ( {
-	method: { key: methodKey, isConfigured, loading, onClick },
-	isEnabled,
-} ) => (
-	<PaymentAction
-		methodKey={ methodKey }
-		hasSetup={ true }
-		isConfigured={ isConfigured }
-		isEnabled={ isEnabled }
-		isRecommended={ true }
-		isLoading={ loading }
-		onSetup={ () => {} }
-		onSetupCallback={ onClick }
-		setupButtonText={ __( 'Get started', 'woocommerce-admin' ) }
-		onManageButtonClick={ () => recordEvent( 'tasklist_payment_manage' ) }
-	/>
-);
-
-export const WCPayMethodCard = ( { method, isEnabled } ) => {
-	const { description } = method;
+export const WCPayMethodCard = ( { suggestion } ) => {
+	const { description, key } = suggestion;
+	const paymentGateway = useSelect( ( select ) => {
+		return (
+			select( PAYMENT_GATEWAYS_STORE_NAME ).getPaymentGateway( key ) || {}
+		);
+	} );
+	const {
+		id,
+		enabled: isEnabled,
+		needs_setup: isConfigured,
```

Comment:
`isConfigured` should actually be the inverse of `needs_setup`.

Reply:
Updated these all to use `needsSetup` instead of `isConfigured` to be more in line with the source.